OSDOCS-12183-rework: rework of changes in delete Kustomize manifest resources (#84633)
openshift/openshift-docs: microshift_running_apps/microshift-deleting-resource-manifests.adoc

:_mod-docs-content-type: ASSEMBLY
[id="microshift-deleting-resource-manifests"]

> = Deleting or upgrading Kustomize manifest resources

Reviewer: Same comment about terminology:
```suggestion
= Deleting or updating Kustomize manifest resources
```

Author: done

include::_attributes/attributes-microshift.adoc[]
:context: microshift-deleting-resource-manifests

toc::[]

{microshift-short} supports the deletion of manifest resources in the following situations:

* Manifest removal: Manifests can be removed when you need to completely remove a resource from the cluster.
* Manifest upgrade: During an application upgrade, some resources might need to be removed while others are retained to preserve data.

When creating new manifests, you can use manifest resource deletion to remove or update old objects, ensuring there are no conflicts or issues.

[IMPORTANT]
====
Manifest files placed in the `delete` subdirectories are not automatically removed and require manual deletion. Only the resources listed in the manifest files placed in the delete subdirectories are deleted.
====

include::modules/microshift-manifests-deletion-overview.adoc[leveloffset=+1]

[id="microshift-examples-of-usecase_{context}"]
== Use cases for manifest resource deletion

> The following explains the use case in which the manifest resource deletion is used.

Reviewer: Suggestion - maybe add a noun here for clarity?
```suggestion
The following sections explain the use case in which the manifest resource deletion is used.
```

Author: Done. This is followed

> The following explains the use case in which the manifest resource deletion is used.

Reviewer: Another suggestion to keep it short and simple, up to you:
```suggestion
The following explains the use cases for manifest resource deletion.
```

Author: above suggestion is followed.

include::modules/microshift-manifests-removal-scenario-rpm.adoc[leveloffset=+2]

include::modules/microshift-manifests-removal-scenario-ostree.adoc[leveloffset=+2]

include::modules/microshift-manifests-upgrade-scenario-rpm.adoc[leveloffset=+2]

include::modules/microshift-manifests-upgrade-scenario-ostree.adoc[leveloffset=+2]

> //[id="additional-resources_microshift-deleting-resource-manifests_{context}"]

Reviewer: Same question about why this ID is commented out

Author: I was trying the levels and had not removed it. Now it is removed.
